modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFExporter.java

```diff
@@ -337,16 +334,16 @@ private void exportArtPartsDataResource(
                 final RTTIObject[] joints = repSkeleton.objs("Joints");
                 for (short i = 0; i < joints.length; i++) {
                     final RTTIObject joint = joints[i];
-                    final Quaternion rotations;
+                    final double[] rotations;
```

Comment:
JOML has quaternions as well. Let's have concrete types instead of generic arrays everywhere.